### DISCUSSION TITLE
Has anyone been banned using this before?

### DISCUSSION THREAD

**Discussion by Sinkcat113** [jwallet/spy-spotify#510] [General]
Wandering if anyone has been banned using this before, Incase I need to make a backup account.

**Comment by zygiu-zygis** [comment]
I don't have any problems with Spytify. This way of sound recording  is probably the same as a normal activity. I have once blocked with other software that somehow gets faster rip ( x2 times or faster than normal playback).

**Comment by jwallet** [comment]
Of all my accounts I never got banned, but dont risk it and create a free account just for this tool
